core/modules/modules.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Provides helper functions for handling user and system modules.
+ *
+ * Written By:
+ * 		Matthew Knox
+ *
+ * License:
+ *		MIT License. All code unless otherwise specified is
+ *		Copyright (c) Matthew Knox and Contributors 2015.
+ */
+
+var loaders         = [require.once('./kassyModule.js')],
+    config          = require('./../config.js'),
+    conflict        = 1;
+
+exports.listModules = function (disabled) {
+    var modules = {};
+    for (var i = 0; i < loaders.length; i++) {
+        var m = loaders[i].listModules(disabled);
+        for (var key in m) {
+            var t = key;
+            while (modules[t]) {
+                t = key + conflict++;
+            }
+            modules[t] = m[key];
+            modules[t].__loaderUID = i;
+        }
+    }
+    return modules;
+};
+
+exports.loadModule = function (module) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529926650/issues/source?bid=3282987&fileBranchId=3282998#l32)